cssom-view-1/Overview.bs

```diff
@@ -1781,46 +1781,76 @@ Document includes GeometryUtils;
 typedef (Text or Element or CSSPseudoElement or Document) GeometryNode;
 </pre>
 
-The <dfn method for=GeometryUtils lt="getBoxQuads(options)|getBoxQuads()">getBoxQuads(<var>options</var>)</dfn> method must run the following steps:
-
+The helper method getCompleteTransform(anchestor) must run the following steps:
```

Comment:
You need to get the element as a parameter, and use `<div algorithm>` for this for example:

```
<div algorithm="get the complete transform">
  In order to <dfn export>get the complete transform</dfn> of an node <var>node</var> relative to a node <var>ancestor</ancestor> ...
</div>

Reply:
I don't get what you mean with this....
getCompleteTransform has a parameter, wich is anchestor